docker/maistra-builder_1.2.Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN curl -sfL https://install.goreleaser.com/github.com/ValeLint/vale.sh -o ./va
     rm -f ./vale.sh
 
 ADD scripts/entrypoint.sh /usr/local/bin/entrypoint
+RUN chmod +x /usr/local/bin/entrypoint
```

Comment:
I don't think you need this. That file already has the executable bit.

Reply:
have you tested it? it failed for me and on prow without this

Reply:
It seems somehow the change I made to that file (chmod +x) got lost. So, if you set this bit on that file and commit it, you won't need this line. This can be done in a follow-up of course.